add-ons/che-stacks/README.md

```diff
@@ -0,0 +1,4 @@
+# Eclipse Che Stacks Addon
+
+This is part of the Che Minishift addon. Please refer to README.md in che addon folder.
```

Comment:
And this is why we should think of a dependency option to allow related add-ons to be selected/enabled/installed.